app/lib/hooks/useLocalStrorage.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react'
+
+export const useLocalStorage = <T>(
```

Comment:
hey @RaiVaibhav can you look at why this might be failing?

![image](https://github.com/deepgram-devs/deepgram-conversational-demo/assets/956290/e89c26b2-6d31-4943-bd76-fec65bc3b8cd)

i think it may be trying to interpret the entire function as JSX because it starts with <T>, it's possible that is in the wrong place?

Reply:
My bad, @lukeocodes kindly check the latest commit I have pushed.